src/extension.ts

```diff
@@ -62,14 +62,31 @@ export async function activate(context: vscode.ExtensionContext) {
     })
   );
 
-  const nodeExplorerProvider = new NodeExplorerProvider(tailscaleInstance, configManager, ssh);
+  // eslint-disable-next-line prefer-const
+  let nodeExplorerView: vscode.TreeView<PeerTree | FileExplorer>;
+
+  function updateNodeExplorerTailnetName(name: string) {
+    nodeExplorerView.title = `Nodes (${name})`;
```

Comment:
Should we maybe have `Tailscale: <name>` -- Or `Tailnet: <name>` -- where the default just says `Tailnet` in package.json. 

Just so it's not almost always `...` suffixed.